test/demo/utils/cardUtils.ts

```diff
@@ -0,0 +1,84 @@
+import {ElementHandle} from 'puppeteer';
+import * as moment from 'moment';
+
+import {Notification} from '../../../src/client';
+
+import {promiseMap} from './asyncUtils';
+import {getToastCards} from './toastUtils';
+import {getCenterCardsByNotification} from './centerUtils';
+
+export interface NotificationCardMetadata {
+    title?: string;
+    body?: string;
+    sourceApp?: string;
+    timeString?: string;
+    icon: string;
+    buttons?: ButtonMetadata[]
+}
+
+export interface ButtonMetadata {
+    title?: string;
+    iconUrl: string;
```

Comment:
Title is optional, but icon is required? Is that right?..

Reply:
Yes.

Reply:
As discussed, added comment making it clear that this is different to `ButtonOptions` from the client